### ISSUE TITLE
[Scenarios - FlightItinerary.Json] - Voice Over is not conveying the complete name for the days under 2 Stops and Non- stop.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7863]
### Target Platforms

iOS

### SDK Version

Version 1.0(2.4.0-beta.22.9.4.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 15.6.1
Application: Adaptive Cards

App Version: Version 1.0(2.4.0-beta.22.9.4.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Scenarios' control and activate it.
6. Navigate to 'FlightItinerary.json' under 'Elements' list item and activate it.
7. Navigate to Date and Time details under 2 stops and observe the issue.

**Actual Result:**
Voice over is conveying incomplete information about the days while navigating to the date and time details under 2 stops and Non stops, Voice over is announcing as **'Tue comma May 30 comma 2017 12 colon 25 PM'**.

**Expected Result:**
Voice over should convey the complete day name information to the user and it should announce as **'Tuesday comma May 30 comma 2017 12 colon 25 PM'**

**Notes:**
Issue is repro throughout application.

**User Experience:**
It will impact the screen reader user as they will not get to know about the day and get confused about the date information.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/190315630-81dce0d9-85f7-406f-a889-1c46a78f13a8.MP4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Now, this issue is tracked using below URL, hence closing this issue.

https://github.com/microsoft/AdaptiveCards-Mobile/issues/41
